README.md

```diff
@@ -27,10 +27,14 @@ Three primary elements make up the CONNECT solution:
 
 History
 -------
+* 4.4 released December 2014
+* 4.3 released March 2014
+* 4.2 released August 2013
+* 4.1 released April 2013
 * 4.0 released February 2013
 * 3.3 released March 2012
 
-For more information, about CONNECT's history see [HISTORY.md](./CONNECT/HISTORY.md)
+For more information about CONNECT's history, see [HISTORY.md](./HISTORY.md)
 
 Getting Started
```

Comment:
Everything in the getting started section can be removed, except for a quick summary of the build steps.  Everything else is covered by a link to the base installation instructions.